snippets/rust.snippets

```diff
@@ -53,39 +74,33 @@ snippet crate
 	#![comment = "${4:Comment.}"]
 	// Specify the output type
 	#![crate_type = "${5:lib}"]
-snippet allow
-	#[allow(${1:unused_variable})]
-snippet feat
-	#![feature(${1:macro_rules})]
```

Comment:
Changed the default tabstop text because macros are no longer feature-gated, instead used something that is unlikely to change from this status for a long time or maybe ever.